wagtail/admin/templates/wagtailadmin/shared/header.html

```diff
@@ -7,20 +7,26 @@
     - `subtitle` - Within the `h1` tag but smaller
     - `search_url` - if present, display a search box. This is a URL route name (taking no parameters) to be used as the action for that search box
     - `query_parameters` - a query string (without the '?') to be placed after the search URL
+    - `classname` - if present, adds classname to the header class list. This will be the class/classes added to the header class list
     - `icon` - name of an icon to place against the title
+    - `avatar` - if present, display an 'avatar' in place of icon. This is the URL to be used as the img src for avatar 
     - `merged` - if true, add the classname 'merged'
     - `action_url` - if present, display an 'action' button. This is the URL to be used as the link URL for the button
     - `action_text` - text for the 'action' button
     - `action_icon` - icon for the 'action' button, default is 'icon-plus'
 
 {% endcomment %}
-<header class="header {% if merged %}merged{% endif %} {% if search_form %}hasform{% endif %}">
+<header class="{% classnames "header" classname merged|yesno:"merged," search_form|yesno:"hasform," %}">
     {% block breadcrumb %}{% endblock %}
     <div class="row">
         <div class="left">
             <div class="col">
                 <h1 class="header__title">
-                    {% if icon %}{% icon name=icon %}{% endif %}
+                    {% if icon %}
+                        {% icon class_name="w-header__icon" name=icon %}
```

Comment:
```suggestion
                        {% icon class_name="w-header__glyph" name=icon %}
```

Having a generic class for the icon and avatar - we can make styles a bit easier to follow.

Reply:
That'll be great!